src/utils/optimizely.js

```diff
@@ -0,0 +1,59 @@
+export const EVENTS = {
+  ENROLLMENT_CLICK: 'enterprise_learner_portal_enrollment_click',
```

Comment:
Event names are a little different from project, experiment, and variant IDs.  We can take the approach of giving all of the same events across environments (e.g. learner clicks the “Enroll” button) the same API name (e.g. `enterprise_learner_portal_enrollment_click`).  Then we could store event identifiers in code instead of environment variables.  In doing so, we wouldn’t have to manage config across environments, we’d just have to ensure that the same API name was consistent across Optimizely custom events in each environment (and we’d have to manage those names in Optimizely regardless).

The only drawback to this is inconsistency - we'd be managing Optimizely identifiers of most types as environment variables, but event identifiers as Javascript constants.